test/navigationLibrary/StandardRoute/StandardRouteWaypointModel.spec.js

```diff
@@ -69,48 +70,65 @@ ava('.toWaypointModel() returns a new instance of a WaypointModel', t => {
     t.true(result instanceof WaypointModel);
     t.true(result.name === model.name.toLowerCase());
     t.true(_isArray(result.relativePosition));
-    t.true(result.altitudeRestriction === 8000);
-    t.true(result.speedRestriction === 250);
+    t.true(result.altitudeMaximum === -1);
+    t.true(result.altitudeMinimum === 8000);
+    t.true(result.speedMaximum === 250);
+    t.true(result.speedMinimum === 250);
 });
 
-ava('._parseWaypointRestrictions() extracts alititude and speed restrictions from a waypointRestrictions string', t => {
-    const model = new StandardRouteWaypointModel(ROUTE_WAYPOINT_MOCK);
-
-    model._parseWaypointRestrictions(RESTRICTIONS_MOCK);
-
-    t.true(model._altitude === 8000);
-    t.true(model._speed === 250);
+ava('._applyRestrictions() extracts all restrictions when ranged restrictions are used', (t) => {
+    const modelWithAltitude = new StandardRouteWaypointModel(['BAKRR', 'A100+|A140-']);
+    const modelWithSpeed = new StandardRouteWaypointModel(['BAKRR', 'S210+|S250-']);
+    const modelWithAltitudeAndSpeed = new StandardRouteWaypointModel(['BAKRR', 'A100+|A140-|S210+|S250-']);
```

Comment:
I'd like to see these three cases extracted out into 3 separate tests.  And, I can't tell from the delta here, but you may also need to add a test for the case of `['BAKRR']` when there are no restrictions